applications/stepswitch/src/stepswitch_originate.erl

```diff
@@ -71,6 +71,7 @@ start_link(Endpoints, OffnetReq) ->
 %%                     {stop, Reason}
 %% @end
 %%--------------------------------------------------------------------
+-spec init([kz_json:objects() | kapi_offnet_resource:req()]) -> {'ok', state()}.
```

Comment:
In specs, like types, lists need union (`|`) rather than `,` (and if you try to put `,`s there it wouldn't compile).
I tried as much as I could to make the init lists of gen_* modules "look like" the actual list of arguments, but in the end it's just a trick to help documentating and/or readability.